03-specifying-sample-designs.Rmd

```diff
@@ -49,11 +50,11 @@ The general process for estimation in the {srvyr} package is to:
 
 4. Within `summarize()`, specify variables to calculate, including means, totals, proportions, quantiles, and more
```

Comment:
I think this needs to be: 
4. Analyze the data with survey specific functions.

Step 4 can be either summarize functions (in e.g., ch5) or models which are not within summarize.

Reply:
Agreed - need to make more generic - could be modeling, using survey_count, or functions within summarize.